deploy/ansible/Makefile

```diff
@@ -21,7 +21,7 @@ NAMES := $(YAML:%.yaml=%)
 .PHONY: $(NAMES)
 default: prepare_packages install
 
-install: init_once riak elasticsearch fluentbit datalayer sandbox management nginx
+install: init_once installnvidiadocker riak elasticsearch fluentbit datalayer frontend sandbox management nginx
```

Comment:
I think 'frontend' component does not exist anymore.

What happens if the host does not have any Nvidia GPUs? Will the 'installnvidiadocker' still succeed?